packages/editor-ui/src/components/ParameterOptions.vue

```diff
@@ -30,7 +30,6 @@ import Vue, { PropType } from 'vue';
 import { isValueExpression, isResourceLocatorValue } from '@/utils';
 import { useNDVStore } from '@/stores/ndv';
 import { mapStores } from 'pinia';
```

Comment:
Nitpick: Since you've replaced `ndvStore` calls we could remove these imports and mapping of the store